testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/OidcBaseTest.java

```diff
@@ -54,6 +62,7 @@
 import org.wildfly.security.jose.util.JsonSerialization;
 
 import io.restassured.RestAssured;
+import org.wildfly.test.integration.elytron.oidc.client.subsystem.SimpleServletWithScope;
```

Comment:
This should be with the org imports.

Reply:
Thanks @fjuma 
I have addressed these comments.